app/apollo/resolvers/channel.js

```diff
@@ -405,6 +405,25 @@ const channelResolvers = {
           }
         } ) );
 
+        // Create output for graphQL plugins
+        const versionFind = await models.DeployableVersion.find({org_id});
+        const versionObjs = _.map(versionFind, (version)=>{
+          return {
+            name: version.name,
+            uuid: version.uuid,
+          };
+        });
+        const subscriptionFind = await models.Subscription.find({org_id});
+        const subscriptionObjs = _.map(subscriptionFind, (subscription)=>{
+          return {
+            name: subscription.name,
+            uuid: subscription.uuid,
+          };
+        });
+
+        // Allow graphQL plugins to retrieve more information
+        context.pluginContext = {channel: {name: newChannelObj.name, uuid: newChannelObj.uuid}, versions: versionObjs, subscriptions: subscriptionObjs};
```

Comment:
For consistency with other actions, the pluginContext should be
```suggestion
        context.pluginContext = { name: newChannelObj.name, uuid: newChannelObj.uuid, versions: versionObjs, subscriptions: subscriptionObjs};
```
(object should have name and uuid top level attributes for the channel, not in a sub attribute).

Reply:
NVM, I see now that the other resolvers now follow the same pattern